docs/src/remote-wallet/ledger.md

```diff
@@ -72,42 +72,29 @@ To fix, check the following:
 3. On your computer, run:
 
 ```text
-solana-keygen pubkey usb://ledger
+solana pubkey usb://ledger
```

Comment:
Should this be `solana address` ?  If not it was right before.  Or this is a feature request :slightly_smiling_face:

Reply:
Whoops should've dropped this one... it was corrected in a later commit